advanced_functionality/multi_model_linear_learner_home_value/linear_learner_multi_model_endpoint_inf_pipeline.ipynb

```diff
@@ -0,0 +1,1302 @@
+{
```

Comment:
<p>Is <code>print</code>  helpful? Does it make sense to annotate the printed value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>Changing this to print("Endpoint arn is :", endpoint_resource_arn)</p>